csharp/ql/src/experimental/Security Features/campaign/Solorigate/ModifedFnvFunctionDetection.ql

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Looks like this file name contains a typo; it should probably be: ModifiedFnvFunctionDetection.ql (note the missing *i*)

Reply:
I think I fixed it somewhere else, It looks fine on my machine, but I can also blame my eyes. Please let me know if this file still has the typo and I will re-type the filename